R/graded.R

```diff
@@ -564,25 +564,26 @@ grade_if_equal <- function(
   local_options_waldo_compare()
 
   compare_msg <- tryCatch(
-    waldo::compare(x, y, tolerance = tolerance),
+    {
+      time_limit <- knitr::opts_chunk$get("exercise.timelimit") %||%
+        gradethis_default_learnr_options$exercise.timelimit
+      time_limit <- time_limit * 0.8
```

Comment:
I think we should maybe have a gradethis-specific option whose default is `exercise.timelimit * 0.8`?

Reply:
That makes sense! Because naming things is hard™, did you have an idea in mind? Would it be something like `gradethis.timelimit`, or more specific like `gradethis.waldo.timelimit`?

Reply:
I'm liking the specific `gradethis.waldo.timelimit` since this is a time limit set for the `waldo::compare()`? But, if you wanted to be more generic maybe `gradethis.compare.timelimit` (in case  we didn't want to use `waldo` anymore or something)

Reply:
I implemented it in [7986e76](https://github.com/rstudio/gradethis/pull/329/commits/7986e763388815b4de0547e365ed721d804630ef) as `diff.timelimit` (just to keep our options open in case we want to use `diffobj` or something instead of `waldo` in the future)

Reply:
My naming convention here (and in general) for options that serve as default values of arguments is to use `<package>.<function>.<argument>`. This is a little tricky since we don't export `grade_if_equal()`, so my preference is `gradethis.compare_if_equal.timelimit`. 

Or, since this isn't directly attached to an argument of an exported function, we could go for symmetry with the learnr's `exercise.timlimit` option and use `gradethis.compare_timelimit`.